src/lm_robust_helper.cpp

```diff
@@ -447,7 +463,7 @@ List lm_variance(Eigen::Map<Eigen::MatrixXd>& X,
 
           } else {
 
-            if (se_type == "CR2") {
+            if (se_type == "CR2" || se_type == "CR3") {
```

Comment:
Choose if you want 
 
    ((se_type == "CR2") || (se_type == "CR3"))

    (se_type == "CR2" || se_type == "CR3")

    (se_type %in% c("CR2", "CR3"))

and be consistent throughout.